src/commands/test/lib/run-tests-command.ts

```diff
@@ -199,4 +205,16 @@ export class RunTestsCommand extends AppCommand {
     let parsedParameters = parseTestParameters(this.testParameters);
     _.merge(manifest.testFramework.data, parsedParameters || {});
   }
+
+  private async processLanguage(): Promise<void> {
+    if (!this.language) {
+      return;
+    }
+    for (let index in this.testParameters) {
+      if (this.testParameters[index].indexOf("language=") === 0) {
+        throw new Error("Argument '--language' cannot be used with a 'language' test parameter");
+      }
+    }
+    this.testParameters.push("language=" + this.language);
```

Comment:
@jonstoneman this doesn't actually show that the code is working because given a locale, say `zn_CN`, Test Cloud will default to language `zn`.  In this case that language doesn't actually exist which is why we wanted to create the `--language` parameter - so we can override the default.

You should test this by invoking like this:

```
--locale "zh_CH" --language "zh-Hans"
```

and verify in `output.log` the log lines where we're setting locale

```
2017-02-13 17:49:01 +0100 Setting lang/locale: language= zh-Hans, locale=zh_CN
POST /sessions/token-xxxx/core-ios/set_locale => 200
```